src/app.py

```diff
@@ -4,8 +4,8 @@
 import pandas as pd
 
 # Read in data
-traits_raw_df = pd.read_csv("breed_traits.csv")
-breed_rank_raw_df = pd.read_csv("breed_rank.csv")
+traits_raw_df = pd.read_csv("data/breed_traits.csv")
```

Comment:
Does this actually work?  The file here is `src/app.py`.  I think the path should be `../data/breed_traits.csv`.  Please correct me if I'm wrong.

Reply:
It worked for me when running the app from the root directory with the command `python src/app.py`

Reply:
Yep, it worked for me when running the app from the root directory (the earlier version did not).